frontend/src/pages/DocumentItemsPage.jsx

```diff
@@ -0,0 +1,324 @@
+import { useMemo, useState, useEffect } from "react";
+import { useLocation, useNavigate, useParams } from "react-router-dom";
+import DynamicTable from "@/components/table/DynamicTable";
+import PageTitle from "@/components/shared-ui/PageTitle";
+import IsError from "@/components/shared-ui/IsError";
+import { Spinner } from "@/components/ui/spinner";
+import { Button } from "@/components/ui/button";
+import { useDocumentItems } from "@/queries/documentItems";
+
+// helperi iz starog koda
```

Comment:
This helper should be moved to utils and imported here.
1. Create frontend/src/utils/numberHelpers.js
2. Export n2 and r2 functions
3. Import in DocumentItemsPage.jsx